<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QWidget" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>322</width>
    <height>221</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>322</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>222</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OK">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>222</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QLabel" name="fitModeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Default fit mode:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="fitModeComboBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ALL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WIDTH</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NORMAL</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="scalingQualityLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Scaling quality: </string>
     </property>
    </widget>
    <widget class="QComboBox" name="scalingQualityComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Better quality</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Better speed</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="fullscreenCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Open in fullscreen</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Loader</string>
    </attribute>
    <widget class="QSlider" name="cacheSlider">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>512</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="preloaderCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Preloads next image in directory for faster scrolling.</string>
     </property>
     <property name="text">
      <string>Enable preloader</string>
     </property>
    </widget>
    <widget class="QLabel" name="cacheLabel2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cacheLabel1">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>When preloader used, ram usege may exceed this setting.</string>
     </property>
     <property name="text">
      <string>Cache size, MB:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>222</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cacheSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cacheLabel2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>applySettingsAndClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>applySettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>applySettingsAndClose()</slot>
  <slot>applySettings()</slot>
 </slots>
</ui>
